app/services/imports/import_report_service.rb

```diff
@@ -84,5 +84,69 @@ def generate_unassigned_logs_report(report_suffix)
 
       @logger.info("Unassigned logs report available in s3 import bucket at #{report_name}")
     end
+
+    def generate_missing_answers_report(report_suffix)
+      create_missing_answers_report(report_suffix, LettingsLog)
+      create_missing_answers_report(report_suffix, SalesLog)
+    end
+
+    def create_missing_answers_report(report_suffix, log_class)
+      class_name = log_class.name.underscore.humanize
+      Rails.logger.info("Generating missing imported #{class_name}s answers report")
+
+      unanswered_question_counts, missing_answers_example_sets = process_missing_answers(log_class)
+
+      report_name = "MissingAnswersReport#{log_class.name}_#{report_suffix}.csv"
+      @storage_service.write_file(report_name, BYTE_ORDER_MARK + missing_answers_report(unanswered_question_counts))
+
+      examples_report_name = "MissingAnswersExamples#{log_class.name}_#{report_suffix}.csv"
+      @storage_service.write_file(examples_report_name, BYTE_ORDER_MARK + missing_answers_examples(missing_answers_example_sets))
+
+      @logger.info("Missing #{class_name}s answers report available in s3 import bucket at #{report_name}")
+    end
+
+    def process_missing_answers(log_class)
+      unanswered_question_counts = {}
+      missing_answers_example_sets = {}
+
+      log_class.where.not(old_id: nil).where(status: "in_progress").each do |sales_log|
```

Comment:
this should just be `log` not `sales_log`

Reply:
good point!